x-pack/legacy/plugins/reporting/server/lib/export_types_registry.js

```diff
@@ -23,10 +23,12 @@ function scan(pattern) {
 
 const pattern = resolve(__dirname, '../../export_types/*/server/index.[jt]s');
 async function exportTypesRegistryFn(server) {
+  const logger = LevelLogger.createForServer(server, ['reporting', 'export_types_registry']);
```

Comment:
There's a gotcha happening here, that anything that is `oncePerServer` can only take 1 argument, so I can't pass `server` and a `logger` object as well. I didn't spend much time on tackling that.